src/themes/hugo-theme-learn/layouts/shortcodes/code.html

```diff
@@ -0,0 +1 @@
+{{ with .Get 0}}<pre>{{end}}<code{{ with .Get 0}} class="hljs {{.}}"{{end}}>{{ .Inner }}</code>{{ with .Get 0}}</pre>{{end}}
```

Comment:
This is handled natively by Hugo's `highlight`: https://gohugo.io/content-management/shortcodes/#highlight

Reply:
I know but the `highlight` doesn't display the same way as when we use the `` I tried to configure it in hugo, I tried to recompile the hjls theme in the assets and several other things, but I wasn't able to change its display

![Capture d’écran 2020-04-22 à 18 21 39](https://user-images.githubusercontent.com/13801017/80007500-9bc52100-84c6-11ea-8349-be60a0c398d7.png)

Because, for an unknown reason (to me at least), it adds a hard styled `<pre color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4>` And since it's a built-in shortcode we can't change the source

So I took advantage of the presence of hjls in the theme and took a shortcut: to create a shortcode (joke 😅)

If you can find how to correctly set this with `highlight` I'm up for it but I already lost two hours trying unsuccessfully so I wanted to move on